Makefile

```
@@ -191,3 +191,6 @@ else
 	@echo "Please provide a service name using `service=[service name]` or `SERVICE=[service name]`"
 endif
 
+.PHONY: reactnative-build-android
+reactnative-build-android:
```

rename this to `build-reactnative-android` to be consistent with our other make target names

```suggestion
.PHONY: build-reactnative-android
build-reactnative-android:
```

---

when with `build-react-native-android` to match the other hyphenation: https://github.com/open-telemetry/opentelemetry-demo/pull/1781/commits/cd47bbc74767c0c8b3a25147a30197c80c7873c4